satellite/gracefulexit/chore_test.go

```diff
@@ -66,6 +66,11 @@ func TestChore(t *testing.T) {
 		require.NoError(t, err)
 		nodeIDs := make(storj.NodeIDList, 0, len(exitingNodes))
 		for _, exitingNode := range exitingNodes {
+			// exiting node should not be disqualified
+			node, err := satellite.Overlay.Service.Get(ctx, exitingNode.NodeID)
+			require.NoError(t, err)
+			require.Nil(t, node.Disqualified)
```

Comment:
nit - maybe this is okay for a sanity check but I'm not sure if this needs to be added to TestChore since I don't think the test is currently changing nodes' reputations or doing something that would result in a dq'd node?

Reply:
I think since the chore is running on an interval, it would mean that whenever the chore starts running, it should never get DQed node for that round. Should I add a wait and trigger the chore again to see if it works in that way?

Reply:
I agree with Nat. This test is not doing anything that would dq a node for any reason so I don't see a need to check this. @VinozzZ could you clarify what you are saying in your comment above? I don't think any nodes will be dqed in this test so waiting on the chore shouldn't make a difference.

Reply:
The only change I made for the chore is that the `GetExitingNode` should not get DQed node, which is why I added this test

Reply:
I guess what I was asking is that we should be able to safely assume that *no* nodes are DQed in this test at any point, so why even bother with the check?

Reply:
I see. That makes sense. I did it for sanity check. I can remove it for sure